Makefile

```diff
@@ -77,6 +79,5 @@ ansible-build:
 # Provision Wordpress and the MySQL server using ansible
 .PHONY: provision-wordpress
 provision-wordpress:
-	export ENVIRONMENT="${app_env}"
-	export AWS_PROFILE="${aws_profile}"
+	sed "s/AWS_REGION_TO_REPLACE/${aws_region}/g" "${PWD}/platform/stateful-application/ansible/inventory/${app_env}/.inventory_aws_ec2.yml" > "${PWD}/platform/stateful-application/ansible/inventory/${app_env}/inventory_aws_ec2.yml"
```

Comment:
1 - What happen we want to use another region? it would be better to use environment variables than modify a file?
2 - PWD is not an environment variable defined always, is this executed on a container where we know for sure that it exists?

Reply:
1) Those two variables were deleted as they had been passed to the [Ansible command](https://github.com/jrsalgado/nstsk/blob/fix-region-hardcoded/Makefile#L34-L35) . This is a workaround as the [aws_ec2](https://docs.ansible.com/ansible/latest/collections/amazon/aws/aws_ec2_invent) doesn't allow to specify the region using a command line parameter or an environment var.

2) This code is executed locally and the updated filesystem, with the newly generated file. This passed to the Ansible container to run the provisioning, so yes it will be available in `/ansible`.